keel/api/v1/eligibility_calculator/views.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework.views import APIView
+from rest_framework import status
+from rest_framework.response import Response
+
+from .serializers import EligibilityResultsSeriaizer
+from keel.eligibility_calculator.models import EligibilityResults
+from keel.leads.models import CustomerLead
+
+class EligibilityResultsView(APIView):
+
+    def post(self, request, format='json'):
+        serializer = EligibilityResultsSeriaizer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        
+        valid_data = serializer.validated_data
+
+        # get name and email from validated data
+        name = valid_data['data']['name']
+        email = valid_data['data']['email']
```

Comment:
because we have validated the data in the serialiser , not the email and name fields
always perform a get on dictionary...
else it will throw exception in runtime...